app/models.py

```diff
@@ -1168,7 +1168,7 @@ class Brief(db.Model):
     clarification_questions = db.relationship(
         "BriefClarificationQuestion",
         order_by="BriefClarificationQuestion.published_at",
-        lazy='joined'
+        lazy='select',
```

Comment:
If we keep this as joined, it is not a lazy join but is actually eagerly loaded, meaning we will always be joining the clarification questions table which we don't want to be doing. By changing it to select, it is lazily loaded.

By changing `Brief.serialize` to not access `clarification_questions` by default it means we won't trigger a call to the clarification questions table. So at the moment, only the `get_brief` and `add_clarification_questions` will load clarification questions, and will do so lazily, which will do the N+1 queries. We could add db options to those individual routes to override the behaviour and eagerly load them just for those routes?

Alternatively, we could keep the eager loading and add db options to all the routes that we don't want them to load clarification questions (I think this is a larger number than of those that should load clarification questions).